examples/echo_server/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "chainlink": "github:smartcontractkit/chainlink",
```

Comment:
We can remove this dependency since it's now the repo we're in. Doesn't make sense to have a dependency on ourself in `package.json`. Removing it here should also remove it from `yarn.lock`.